src/main/scala/com.snowplowanalytics.dataflow/streaming/StreamingCountsApp.scala

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.dataflow.streaming
+
+// Config
+import com.typesafe.config.ConfigFactory
+
+/**
+ * The entry point class for the Dataflow Streaming Application.
+ */
+object StreamingCountsApp {
+
+  def main(args: Array[String]) {
```

Comment:
Procedure syntax is deprecated. This should be `def main(args: Array[String]): Unit = {`.